### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by TetianaZinchenko**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <script type="module" src="./index.js"></script>
   </head>
   <body>
+    <include src="./partials/header.html"></include>
```
Comment (AlexStanley) [inline src/index.html:17]: Hello Tetiana

File: src/partials/header.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<h1>Наша команда</h1>
```
Comment (AlexStanley) [inline src/partials/header.html:1]: Good team!

File: src/index.html (hunk 1 above)
Comment (TetianaZinchenko) [reply src/index.html:17]: Everything is OK

File: src/partials/header.html (hunk 2 above)
Comment (TetianaZinchenko) [reply src/partials/header.html:1]: Everything is OK

**Comment by AlexStanley** [comment]
Everything had been done perfectly!

**Review by AlexStanley — APPROVED** [review]

**Pull request merged**